packages/manager/modules/nasha/src/components/partition/edit-size/edit-size.controller.js

```diff
@@ -31,4 +36,9 @@ export default class NashaComponentsPartitionEditSizeController {
       .then((tasks) => this.close({ tasks, partitionName }))
       .catch((error) => this.close({ error }));
   }
+
+  onClickOnClose() {
```

Comment:
Same as https://github.com/ovh/manager/pull/7762/files#r943141172

Reply:
It's applied.